requirements.txt

```diff
@@ -16,8 +16,7 @@ rosdistro==0.6.2
 rosinstall-generator==0.1.13
 rospkg==1.1.1
 six==1.10.0
-trollius==2.1
+trollius==2.1.post2
 urllib3==1.7.1
 vcstools==0.1.39
-wsgiref==0.1.2
```

Comment:
`wsgiref` is part of the standard library in python3. Bazel uses python3, not 2, so the pypi dependency management broke with this (the pypi `wsgiref` is not python3 compatible). Please feel free to let me know if this is needed for python2 and I can find a workaround.